.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to Mobify Code Style
+
+## Bugs, Feature Requests
+Submit an [issue](https://github.com/mobify/mobify-code-style/issues).
+
+## Pull Requests
+
+1. Make a new branch from `develop`.
```

Comment:
The issue that I currently have with code-style using the Continuous Deployment strategy is that we aren't auto-publishing `master` to NPM and that means that someone must manually do it. I've seen this often fall by the wayside and PRs get merged into `master` but not published (maybe they're just tiny doc tweaks) and we get drift between what's out in the wild and what's sitting on `master`. 

For the time being I think that that repo more closely follows Release Deployment, until processes around its publishing are nailed down, even though I agree that Continuous Deployment makes more sense here.

If everyone agrees, I'd volunteer as the release manager for this repo to control deployment. Otherwise we'll have to wire up some continuous publishing or have everyone more on-the-ball with publishing PR changes.